nextcord/member.py

```diff
@@ -745,6 +752,11 @@ async def edit(
             Set this to None to disable their timeout.
 
             .. versionadded:: 2.0
+        flags: :class:`MemberFlags`
```

Comment:
Would it not make sense to be nice to the user like #979, where you do not need to use `flags=MessageFlags(bypasses_verification=True)` but `bypasses_verification=True`